lib/chromedriver.js

```diff
@@ -29,6 +30,8 @@ class Chromedriver extends events.EventEmitter {
     this.jwproxy = new JWProxy({server: this.proxyHost, port: this.proxyPort});
     this.verbose = verbose;
     this.logPath = logPath;
+    this.adb = new ADB();
```

Comment:
Now I use appium-adb to clean adb forward resources. But appium-adb still has no fowardList method. I think I should pull a request to add that method. :-P